windows/scripts/Install-DotNetSdk.ps1

```diff
@@ -1,6 +1,4 @@
-choco install dotnetcore-sdk --version 2.1.302
-choco install dotnetcore-sdk --version 2.1.402
-choco install dotnetcore-sdk --version 2.1.403
-choco install dotnetcore-sdk --version 3.0.100
-choco install dotnetcore-sdk --version 3.1.201
-choco install dotnet-sdk --version 5.0.100
+choco install dotnetcore-sdk --version 2.1.815
+choco install dotnetcore-sdk --version 3.0.103
+choco install dotnetcore-sdk --version 3.1.408
+choco install dotnet-sdk --version 5.0.202
```

Comment:
shouldn't this be 5.0.3? `If you're using .NET 5.0, you should download and install Runtime 5.0.3 or SDK 5.0.103 (for Visual Studio 2019 v16.8)` from https://github.com/dotnet/announcements/issues/176

Reply:
ah sorry, 5.0.202 is newer than  5.0.103 :)

Reply:
Yes, it's equivalent of 5.0.5. See here: https://dotnet.microsoft.com/download/dotnet/5.0